src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/imports/ImportOrderCheck.xml

```diff
@@ -40,7 +40,7 @@
  &lt;a href="https://en.wikipedia.org/wiki/ASCII#Order"&gt;ASCII sort order&lt;/a&gt;.
  It affects both type imports and static imports.</description>
             </property>
-            <property default-value="" name="groups" type="java.util.regex.Pattern[]">
+            <property default-value="" name="groups" type="java.lang.String[]">
```

Comment:
Has it been discussed what impact this has on 3rd party tools like eclipse-cs?

Reply:
It will be declared breaking compatibility, I hope eclipscs will not be actually affected.

Reply:
So we are not going to do any analysis or warning?

Reply:
yes, let move forward, and just declare breaking changes.